diagnostic.md

```diff
@@ -6,72 +6,87 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```bash
-// your response here
+  A backend allows data to be stored and persist regardless of what is happening
+  on the client side of an application. It also allows different clients using
+  and app to interact with each other
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```bash
-// your response here
+  The model is used to get data from a database
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```bash
-// your response here
+  The controller communicates with the model
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```bash
-// your response here
+  Because it is 2016 and we make single-page apps
 ```
 
 What does C.R.U.D stand for?
 
 ```bash
-// your response here
+  Create
+  Read
+  Update
+  Destroy
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```bash
-// your response here
+  The Model uses CRUD actions to interact with the database
 ```
 
 A user action fires a `GET` request for `person/1`. Explain in detail each step
 required for data to be returned to the client. (bullet points or ordered list)
 
 ```bash
-// your response here
+  - The GET request is routed to the correct controller for people
+  - The people controller sends the GET request to the correct model
```

Comment:
the request doesn't get sent the the model, the controller interprets the request relays it's needs to the model